### PR TITLE
Spotify clone project

### PR DESCRIPTION
**Pull request by devanshtivari**
Issue number #152 resolved.
Created a Spotify clone and added files in the respective folders.
image

Screenshot of the porject

### CODE REVIEW

File: Public/spotify-clone.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Spotify - Your favourite music is here</title>
+        <link rel="stylesheet" href="style.css">
```
Comment (Astrodevil) [inline Public/spotify-clone.html:8]: linkage is wrong make it correct.

File: Public/spotify-clone.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            <!-- fontawesome icons -->
+            <i class="fas fa-2x fa-step-backward" id="previous" style="padding: 10px;"></i>
+            <i class="far fa-2x fa-play-circle" id="masterPlay" style="padding: 10px;"></i>
+            <i class="fas fa-2x fa-step-forward" id="next" style="padding: 10px;"></i> 
+        </div>
+        <div class="songInfo">
+            <img src="playing.gif" width="42px" alt="" id="gif"> <span id="masterSongName">Warriyo - Mortals [NCS Release]</span>
+        </div>
+    </div>
+    <script src="script.js"></script>
```
Comment (Astrodevil) [inline Public/spotify-clone.html:88]: linkage is wrong make it correct. you have to specify path

**Comment by devanshtivari** [comment]
Updated every path and made changes in script to enhance interactivity

**Comment by Astrodevil** [comment]
Please add that songs, everything else is fine

**Comment by devanshtivari** [comment]
Added songs and updated their paths in spotify-clone.js

**Pull request merged**